src/almanack/metrics/data.py

```diff
@@ -214,29 +214,29 @@ def compute_repo_data(repo_path: str) -> None:
 
         # Return the data structure
         return {
-            "repo_path": str(repo_path),
-            "number_of_commits": len(commits),
-            "number_of_files": len(file_names),
-            "time_range_of_commits": (first_commit_date, most_recent_commit_date),
-            "readme-included": file_exists_in_repo(
+            "repo-path": str(repo_path),
+            "repo-commits": len(commits),
+            "repo-file-count": len(file_names),
+            "repo-commit-time-range": (first_commit_date, most_recent_commit_date),
+            "includes-readme": file_exists_in_repo(
```

Comment:
i'm wondering why some of these are prefixed with `repo` and others aren't? Does it make sense to keep naming consistent? (i can totally see the argument that adding the prefix to these others is redundant)

Reply:
Thanks, this is a great point! I'll err on the side of over-description to keep all names with a "subject" of `repo` (in case we deviate from this pattern to focus also on other areas of the software).